### PR TITLE
hmi try/except TimeoutException

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
plus python3 compatibility issue

### CODE REVIEW

**Comment by Rayman** [comment]
@MatthijsBurgh does this solve your issue with the lights?

**Comment by MatthijsBurgh** [comment]
Yep it does

File: robot_skills/src/robot_skills/api.py [hunk 1]
```diff
@@ -35,7 +35,13 @@ def query(self, description, grammar, target, timeout=10):
         if callable(self._pre_hook):
             self._pre_hook()
 
-        answer = self._client.query(description, grammar, target, timeout)
+        try:
+            answer = self._client.query(description, grammar, target, timeout)
+        except TimeoutException as e:
+            rospy.logdebug(e)
```
Comment (Rayman) [inline robot_skills/src/robot_skills/api.py]: The log is not needed because you re-reraise, the exception can be logged on a higher level

Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/api.py]: Done, please review again.

**Pull request merged**